app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script.rb

```diff
@@ -20,9 +20,7 @@ def self.stack_type
     "Job"
   end
 
-  def supports_limit?
-    true
-  end
+  supports :limit
```

Comment:
@chessbyte `limit` doesn't appear to be a valid supports feature.  It is a little generic, wonder if we should call this something more helpful

Reply:
Any suggestions?  We really should get an understanding of what this does based on the callers, I guess.

Reply:
limit is the ansible feature - it's a way to further restrict playbooks to specific hosts.  So if a playbook is declared `hosts: all`, you can run it with `--limit datacenter2` to restrict it to just that host.

https://docs.ansible.com/ansible/latest/user_guide/intro_patterns.html#patterns-and-ansible-playbook-flags

Reply:
So maybe `supports :hosts_limit` ?

Reply:
actually, we don't use the terms hosts in configuration manager...Not sure what the alternative term is offhand (configured_systems?), but we may want to to be that `<that-term>_limit`

Reply:
If we can scope the feature to just `ConfigurationScript` like @chessbyte's supports feature mixin refactoring PR I'd be good with `hosts_limit`.

Having that in the main QUERYABLE_FEATURES hash it would be too generic IMO

Reply:
I am envisioning a world where the only globally supported features are CRUD+refresh - or maybe even less than that ;-)